mdx2/command_line/scale.py

```
@@ -2,328 +2,294 @@
+        # Apply mca2020 shortcut to enable all models
+        if self.mca2020:
+            self.scaling.enable = True
+            self.detector.enable = True
+            self.absorption.enable = True
+            self.offset.enable = True
+
+        # Check for duplicate input files
+        # Note: Same filename in different directories is valid (e.g., /path1/data.nxs, /path2/data.nxs)
+        # But truly duplicate paths are not allowed (e.g., data.nxs, data.nxs)
+        if len(self.hkl) != len(set(self.hkl)):
+            raise ValueError("Duplicate input files are not allowed")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)